src/sidebar-items.json

```diff
@@ -169,6 +169,17 @@
     "url": "form-layout.html",
     "icon": "file-earmark-medical-fill"
   },
+  {
+    "name": "Form Validation",
+    "icon": "pen-fill",
```

Comment:
Can you change the icon to something like bootstrap icon's `journal-check`?

Reply:
real